doc/nrf/ug_thread_certification.rst

```diff
@@ -0,0 +1,109 @@
+====================
+
+If your solution modifies the stack's certified functionality or if Thread Group rejects your application for certification by inheritance, you need to undergo the full certification procedure at an ATL.
+Only Contributor and Sponsor tier members of Thread Group can apply for certification by an ATL.
+
+Thread Test Harness integration
+*******************************
+
+To certify a Thread product, all certification test cases must be run on the Thread Test Harness software.
+
+A detailed description of how to assemble and configure a Thread Test Bed and run the Thread Test Harness can be found on the `Thread Group's Confluence wiki page`_.
+
+.. note::
+   The following procedure references the nRF52840 Development Kit.
+   The analogical procedure may be used to run the certification using other development kits.
+
+Complete the following steps to run the certification tests:
+
+
+#. Build the certification image.
+
+   The :ref:`ot_cli_sample` sample is used as a base, modifying it with an overlay file.
+
+   .. code-block::
+
+         cd ncs/nrf/samples/openthread/cli/
+         west build -b nrf52840dk_nrf52840 -- -DCONF_FILE="prj.conf harness/overlay-cert.conf"
+
+   .. note::
+      The overlay file selects the precompiled OpenThread libraries by default.
+
+#. Prepare Thread Test Harness.
+
+   a. Copy the provided :file:`ncs/nrf/samples/openthread/cli/harness/nrf52840-ncs.py` file into :file:`C:\\GRL\\Thread1.1\\Thread_Harness\\THCI`.
+
+   b. Copy the provided :file:`ncs/nrf/samples/openthread/cli/harness/nrf52840.jpg` file into :file:`C:\\GRL\\Thread1.1\\Web\\images`.
+
+   c. Edit :file:`C:\\GRL\\Thread1.1\\Web\\data\\deviceInputFields.xml` and prepend this:
+
+      .. code-block::
+
+         <DEVICE name="nRF52840-ncs" thumbnail="nRF52840.jpg" description = "Nordic Semiconductor: nRF52840 (NCS) Baudrate:115200" THCI="nRF52840-ncs">
+            <ITEM label="Serial Line"
+               type="text"
+               forParam="SerialPort"
+               validation="COM"
+               hint="eg: COM1">COM
+            </ITEM>
+            <ITEM label="Speed"
+               type="text"
+               forParam="SerialBaudRate"
+               validation="baud-rate"
+               hint="eg: 115200">115200
+            </ITEM>
+         </DEVICE>
+
+See the following links for more information on OpenThread:
+
+- `OpenThread THCI`_
+- `Openthread acting as a new reference platform`_
+
+Thread Test Harness with nRF52840 DK
+====================================
+
+Thread Test Harness does not correctly identify the PCA10056 Development Kit, based on Nordic Semiconductor's nRF52840 SoC, right out-of-the-box.
+
+Due to a collision of USB PID:VID with another vendor (this is valid only for Nordic Semiconductor development kits with J-Link virtual COM port), Nordic devices are not automatically added to the device list.
+
+To add an nRF52840 device, drag the nRF52840 device and drop it on the configuration page.
```

Comment:
So this should be done within the Thread Test Harness software? @edmont

Reply:
Yes, in its web interface.